Library/Homebrew/test/formula_spec.rb

```diff
@@ -727,7 +727,7 @@ def test
     stub_formula_loader(f1)
 
     python = PythonRequirement.new
-    x11 = X11Requirement.new("x11", [:recommended])
+    x11 = (OS.mac? ? X11Requirement : XorgRequirement).new("x11", [:recommended])
```

Comment:
LGTM! As long as this work makes Linuxbrew and Homebrew codebase closer together, I'm 👍 on that